### DISCUSSION TITLE
Ereignisanzeige

### DISCUSSION THREAD

**Discussion by Nickel1893** [tbnobody/OpenDTU#572] [Q&A]
Gibt es eigentlich irgendwo eine Aufschlüsselung der Ereignisse / Fehler die in der DTU aufgelistet werden was diese bedeuten? Heute z.B.  Fehlernummer 46 unknown.
Vielen Dank vorab

**Comment by teesmokr** [comment]
Kann das sein dass du Kabel am WR vertauscht hast ? Z.B. Plus von einem und Mius vom anderen Modul an einem Tracker ? Das hatte ich nämlich durch einen kleinen Moment der Unachtsamkeit mit gleichem Fehlerbild ;-)

**Reply by Nickel1893** [reply]
Danke fürs feedback. Kabel dürften richtig angeschlossen sein :) Fehler 46 kam bisher nur ein mal.
Hab später topic #590 gefunden, da wird über Fehlercodes diskutiert.

**Comment by sschoeppi** [comment]
Hallo, ich klinke mich hier ein, weil ich ebenfalls eine Frage zur Ereignisanzeige habe. Seit ca. 3 Monaten betreibe ich die Open-DTU erfolgreich. Nur bei der Ereignisanzeige erhalte die falschen Uhrzeiten. Die NTP Einstellungen habe ich gemacht.  Auch die Reihenfolge der Meldungen, also wie sie angezeigt werden, erscheint mir irgendwie unlogisch.
Hat jemand eine Idee was ich falsch konfiguriert haben könnte, bzw. wo man das noch einmal nachlesen kann?
vg Stephan
